### PR TITLE
Add provider configs

### PR DESCRIPTION
**Pull request by jllopes**

### CODE REVIEW

File: codacy/values.yaml [hunk 1]
```diff
@@ -246,6 +246,24 @@ crow:
       port: 5432
     fullnameOverride: codacy-crowdb
 
+githubEnterprise:
```
Comment (paulopontesm) [inline codacy/values.yaml]: Add this on the documentation

Comment (paulopontesm) [reply codacy/values.yaml]: here: https://github.com/codacy/chart/blob/dev/docs/configuration/index.md

File: docs/configuration/index.md [hunk 2]
```diff
@@ -4,61 +4,70 @@ The following table lists the configurable parameters of the Codacy chart and th
...
+| `hotspots-api.hotspotsdb.postgresqlUsername` | Username of the Postgresql server        | `codacy`        |
+| `hotspots-api.hotspotsdb.postgresqlDatabase` | Database name of the Postgresql server   | `hotspots`      |
+| `hotspots-api.hotspotsdb.postgresqlPassword` | Hostname of the Postgresql server        | None            |
+| `hotspots-api.hotspotsdb.host`               | Hostname of the Postgresql server        | None            |
+| `hotspots-api.hotspotsdb.service.port`       | Port of the Postgresql server            | `5432`          |
+| `githubEnterprise.hostname`                  | Hostname of githubEnterprise instance    | None            |
+| `githubEnterprise.protocol`                  | Protocol of githubEnterprise instance    | None            |
+| `githubEnterprise.port`                      | Port of githubEnterprise instance        | None            |
+| `githubEnterprise.isPrivateMode`             | PrivateMode of githubEnterprise instance | None            |
+| `githubEnterprise.disableSSL`                | hasCustomSSL on githubEnterprise instance| None            |
```
Comment (machadoit) [inline docs/configuration/index.md]: Disable certification validation when interacting with Github Enterprise instance

File: docs/configuration/index.md [hunk 3]
```diff
@@ -4,61 +4,70 @@ The following table lists the configurable parameters of the Codacy chart and th
...
+| `activities.activitiesdb.service.port`       | Port of the Postgresql server            | `5432`          |
+| `hotspots-api.hotspotsdb.postgresqlUsername` | Username of the Postgresql server        | `codacy`        |
+| `hotspots-api.hotspotsdb.postgresqlDatabase` | Database name of the Postgresql server   | `hotspots`      |
+| `hotspots-api.hotspotsdb.postgresqlPassword` | Hostname of the Postgresql server        | None            |
+| `hotspots-api.hotspotsdb.host`               | Hostname of the Postgresql server        | None            |
+| `hotspots-api.hotspotsdb.service.port`       | Port of the Postgresql server            | `5432`          |
+| `githubEnterprise.hostname`                  | Hostname of githubEnterprise instance    | None            |
+| `githubEnterprise.protocol`                  | Protocol of githubEnterprise instance    | None            |
+| `githubEnterprise.port`                      | Port of githubEnterprise instance        | None            |
+| `githubEnterprise.isPrivateMode`             | PrivateMode of githubEnterprise instance | None            |
```
Comment (machadoit) [inline docs/configuration/index.md]: Enable private mode when you have that setting enabled on your GithubEnterprise instance

**Pull request merged**